fabric-rendering-v1/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,26 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-rendering-v1",
+  "name": "Fabric Rendering (v1)",
+  "version": "${version}",
+  "environment": "client",
+  "license": "Apache-2.0",
+  "icon": "assets/fabric-rendering-v1/icon.png",
```

Comment:
Does it have an icon? I dont see one?

Reply:
Yes https://github.com/FabricMC/fabric/pull/359/files#diff-ed35d09ef0daa4262750b647c8e67b25

Reply:
Ah yeah, I missed that. Thanks.